GameEngine/Entities/Antilope.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Repository;
+
+namespace GameEngine.Entities
+{
+    /// <summary>
+    /// Antilope entity.
+    /// </summary>
+    public class Antilope : IAnimal
+    {
+        public Antilope()
+        {
+            Id = GenerateId();
+            Type = AnimalType.Antilope;
+            Vision = ConstantsRepository.AntilopeVision;
+            IsDead = false;
+            Letter = ConstantsRepository.AntilopeLetter;
+        }
+
+        public int Id { get; set; }
+        public int CoordinateX { get; set; }
+        public int CoordinateY { get; set; }
+        public AnimalType Type { get; set; }
+        public int Vision { get; set; }
+        public bool IsDead { get; set; }
+        public string Letter { get; set; }
+
+        private static int id = 0;
```

Comment:
Two Id properties ? And as far as I can see Id will always be 1.

Reply:
No, It is increment) I checked it now

Reply:
Done.